frames/src/main/java/com/checkout/frames/screen/billingaddress/billingaddressdetails/BillingAddressDetailsViewModel.kt

```diff
@@ -116,13 +116,13 @@ internal class BillingAddressDetailsViewModel @Inject constructor(
     // Update the country component state if the country is different from current value
     internal fun updateCountryComponentState(
         state: BillingAddressInputComponentState,
-        country: Country,
+        country: Country?,
     ) {
         with(state) {
-            val isCountryValid = country != Country.INVALID_COUNTRY
-            if (isAddressFieldValid.value != isCountryValid)
-                isAddressFieldValid.value = isCountryValid
-            if (addressFieldText.value != country.name) addressFieldText.value = country.name
+            country?.let {
```

Comment:
Shall we warp `with(state)` with `country?.let`? 

```kotlin
country?.let {
    with(state) {
        isAddressFieldValid.value = true
        if (addressFieldText.value != country.name) addressFieldText.value = country.name
    }
}
```

Reply:
Sure